jetty-annotations/src/main/java/org/eclipse/jetty/annotations/AnnotationConfiguration.java

```diff
@@ -491,7 +491,9 @@ public void run()
                 LOG.debug("Scanned {} in {}ms", p.getResource(), TimeUnit.MILLISECONDS.convert(p.getStatistic().getElapsed(), TimeUnit.NANOSECONDS));
 
             LOG.debug("Scanned {} container path jars, {} WEB-INF/lib jars, {} WEB-INF/classes dirs in {}ms for context {}",
-                    _containerPathStats.getTotal(), _webInfLibStats.getTotal(), _webInfClassesStats.getTotal(),
+                    (_containerPathStats==null?-1:_containerPathStats.getTotal()), 
+                    (_webInfLibStats==null?-1:_webInfLibStats.getTotal()), 
+                    (_webInfClassesStats==null?-1:_webInfClassesStats.getTotal()),
```

Comment:
If the stats are only reported here, when debug is turned on, then perhaps they should only be set and updated when debug is also turned on?

Reply:
Done.